packages/api/src/router/github.ts

```diff
@@ -196,19 +218,220 @@ export const githubRouter = createTRPCRouter({
         ),
     ),
 
+    byWorkspaceId: protectedProcedure
+      .input(z.string().uuid())
+      .query(async ({ ctx, input }) => {
+        const internalOrgs = await ctx.db
+          .select()
+          .from(githubOrganization)
+          .where(eq(githubOrganization.workspaceId, input));
+
+        return getOctokit()
+          .apps.listInstallations({
+            headers: {
+              "X-GitHub-Api-Version": "2022-11-28",
+            },
+          })
+          .then(({ data: installations }) =>
+            Promise.all(
+              installations.filter(
+                (i) =>
+                  i.target_type === "Organization" &&
+                  internalOrgs.find((org) => org.installationId === i.id) !=
+                    null,
+              ),
+            ),
+          );
+      }),
+
     list: protectedProcedure
       .input(z.string().uuid())
       .query(({ ctx, input }) =>
         ctx.db
           .select()
           .from(githubOrganization)
+          .leftJoin(
+            githubUser,
+            eq(githubOrganization.addedByUserId, githubUser.userId),
+          )
           .where(eq(githubOrganization.workspaceId, input)),
       ),
 
     create: protectedProcedure
       .input(githubOrganizationInsert)
       .mutation(({ ctx, input }) =>
-        ctx.db.insert(githubOrganization).values(input).returning(),
+        ctx.db.transaction((db) =>
+          db
+            .insert(githubOrganization)
+            .values(input)
+            .returning()
+            .then(takeFirst)
+            .then((org) =>
+              getOctokit()
+                .apps.getInstallation({
+                  installation_id: org.installationId,
+                })
+                .then(async ({ data: installation }) => {
+                  const installationOctokit = new Octokit({
+                    authStrategy: createAppAuth,
+                    auth: {
+                      appId: env.GITHUB_BOT_APP_ID,
+                      privateKey: env.GITHUB_BOT_PRIVATE_KEY,
+                      clientId: env.GITHUB_BOT_CLIENT_ID,
+                      clientSecret: env.GITHUB_BOT_CLIENT_SECRET,
+                      installationId: installation.id,
+                    },
+                  });
+
+                  const installationToken = (await installationOctokit.auth({
+                    type: "installation",
+                    installationId: installation.id,
+                  })) as { token: string };
+
+                  const searchResponse = await installationOctokit.search.code({
+                    q: `org:${org.organizationName} filename:example.ctrlplane.yaml`,
```

Comment:
```suggestion
                    q: `org:${org.organizationName} filename:*ctrlplane.yaml`,
```

Does it support regex? Also yml is also valid yaml